Mathlib/Algebra/Lie/UniversalEnveloping.lean

```diff
@@ -108,9 +108,8 @@ def lift : (L →ₗ⁅R⁆ A) ≃ (UniversalEnvelopingAlgebra R L →ₐ[R] A)
     --   RingQuot.liftAlgHom_mkAlgHom_apply]
     simp only [LieHom.coe_comp, Function.comp_apply, AlgHom.coe_toLieHom,
       UniversalEnvelopingAlgebra.ι_apply, mkAlgHom]
-    dsimp [UniversalEnvelopingAlgebra]
-    rw [RingQuot.liftAlgHom_mkAlgHom_apply]
-    simp only [TensorAlgebra.lift_ι_apply, LieHom.coe_toLinearMap]
+    simp only [UniversalEnvelopingAlgebra, RingQuot.liftAlgHom_mkAlgHom_apply,
+      TensorAlgebra.lift_ι_apply, LieHom.coe_toLinearMap]
```

Comment:
I think the porting note above this wasn't deleted, but I think it should be. Also, is it possible to merge this `simp only` with the one preceding it, or not?

Reply:
The `simp`s cannot be merged: if we unfold `UniversalEnvelopingAlgebra` in the previous call, there is a type mismatch and `simp` cannot continue. So having two `simp` calls is still a regression from the original situation.